packages/mobile/src/account/LinkBankAccountErrorScreen.tsx

```diff
@@ -0,0 +1,69 @@
+import BorderlessButton from '@celo/react-components/components/BorderlessButton'
+import Button, { BtnSizes, BtnTypes } from '@celo/react-components/components/Button'
+import * as React from 'react'
+import fontStyles from '@celo/react-components/styles/fonts'
+import { StyleSheet, Text, View } from 'react-native'
+import { StackScreenProps } from '@react-navigation/stack'
+import { StackParamList } from 'src/navigator/types'
+import { Screens } from 'src/navigator/Screens'
+import { useTranslation } from 'react-i18next'
+import { navigate, navigateBack } from 'src/navigator/NavigationService'
+
+type Props = StackScreenProps<StackParamList, Screens.LinkBankAccountErrorScreen>
+
+function LinkBankAccountErrorScreen({ route }: Props) {
```

Comment:
I neglected to put it in the AC of the ticket (my bad!) but can you make this component also handle IHL errors from [SyncBankAccountScreen.tsx](https://github.com/valora-inc/wallet/blob/main/packages/mobile/src/account/SyncBankAccountScreen.tsx#L38-L49) ?  

Could be done in a subsequent PR or this one, otherwise the PR looks good!